front/app/api/map_config/keys.ts

```diff
@@ -7,15 +7,15 @@ const mapConfigKeys = {
   items: () => [{ ...baseKey, operation: 'item' }],
   item: ({
```

Comment:
Main issue was this old caching bug I introduced in the keys.ts.

I also included some better types and cleanup in this PR as well.